### PR TITLE
implements the ned

### PR DESCRIPTION
**Pull request by kprevas**

### CODE REVIEW

File: deckimport.js [hunk 1]
```diff
@@ -18,13 +17,26 @@ function parseDeck(text, deck, output, sideboard, callback) {
...
+function doRequest(names, counts, deck, output, sideboard, callback, page) {
+    $.get("https://api.magicthegathering.io/v1/cards?page=" + page + "&name=" + names.join("|"),
         function (data) {
             for (i = 0; i < data.cards.length; i++) {
                 var card = data.cards[i];
                 if (counts.hasOwnProperty(card.name.toLowerCase())) {
+                    var color;
+                    if (!card.colorIdentity) {
+                        color = "C";
+                    } else if (card.colorIdentity.length == 1) {
```
Comment (cdanis) [inline deckimport.js]: no gold card support?!?!

File: deckimport.js [hunk 2]
```diff
@@ -18,13 +17,26 @@ function parseDeck(text, deck, output, sideboard, callback) {
...
+                        color = card.colorIdentity[0];
+                    } else {
+                        color = "M";
+                    }
                     deck.push({
```
Comment (cdanis) [inline deckimport.js]: what if deck.push is fatal, though?

File: overlay-ned1.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    <script type="text/javascript" src="cards.js"></script>
+    <script type="text/javascript" src="firebase.js"></script>
+    <script type="text/javascript" src="gamestate.js"></script>
+</head>
+<body>
+
+<svg height="0" width="0">
+    <defs>
+        <clipPath id="clipLeft" clipPathUnits="objectBoundingBox">
+            <path d="M 0 0 L 1 0 L 1 1 C .46 1, .54 0, 0 0"/>
```
Comment (cdanis) [inline overlay-ned1.html]: what the hell is this

Comment (cdanis) [inline overlay-ned1.html]: are you a wizard

Comment (7sempra) [inline overlay-ned1.html]: Yes, he is. I was gonna export it for him but nooooo I'ma hand-code that SVG.

Comment (kprevas) [inline overlay-ned1.html]: I hope sempra notices me

**Pull request merged**